sandbox/acme/src/app/Controllers/AcmeController.php

```diff
@@ -7,7 +7,7 @@
 
 class AcmeController extends BaseController
 {
-    public function testAction()
```

Comment:
This is still under the `main` module, however If you will be checking the kernel, we called the `setActionSuffix` here [Clarity\Providers\Dispather](https://github.com/phalconslayer/framework/blob/master/src/Clarity/Providers/Dispatcher.php#L69)

But this service was extended at [Components\Providers\Dispatcher](https://github.com/phalconslayer/slayer/blob/dev/components/Providers/Dispatcher.php), so just override the `register()` function, or create a function in the kernel as

```
protected function getActionSuffix()
{
    return '';
}
```

and add the same code in the Components\Providers\Dispatcher to apply overriding, but leave it as a commented code instead.